src/hosting.test.js

```diff
@@ -54,22 +54,15 @@ describe("hosting", () => {
   describe("checking a single domain with #check", () => {
     it("use the API instead", async () => {
       const db = await hosting.loadJSON(jsonPath);
```

Comment:
hey @fershad I think if this test is about checking the API, we probably don't need the `const db = await hosting.loadJSON(jsonPath);` bit either, as this test seems to exercise the API only now.